app/src/main/java/cm/aptoide/pt/home/HomeAnalytics.java

```diff
@@ -228,14 +246,33 @@ public void sendAppsChipHomeInteractEvent() {
   public void sendAppsChipInteractEvent() {
     final Map<String, Object> data = new HashMap<>();
     data.put(ACTION, APPS);
-    analyticsManager.logEvent(data, HOME_CHIP_CLICK, AnalyticsManager.Action.CLICK,
+    analyticsManager.logEvent(data, HOME_CHIP_INTERACT, AnalyticsManager.Action.CLICK,
         navigationTracker.getViewName(true));
   }
 
   public void sendGamesChipInteractEvent() {
     final Map<String, Object> data = new HashMap<>();
     data.put(ACTION, GAMES);
-    analyticsManager.logEvent(data, HOME_CHIP_CLICK, AnalyticsManager.Action.CLICK,
+    analyticsManager.logEvent(data, HOME_CHIP_INTERACT, AnalyticsManager.Action.CLICK,
+        navigationTracker.getViewName(true));
+  }
+
+  public void sendChipTapOnMore(String bundleTag, ChipManager.Chip chip) {
+    final Map<String, Object> data = new HashMap<>();
+    data.put(ACTION, TAP_ON_MORE);
+    data.put(BUNDLE_TAG, bundleTag);
+    data.put(CHIP_TAG, chip.getName());
+    analyticsManager.logEvent(data, HOME_CHIP_INTERACT, AnalyticsManager.Action.CLICK,
+        navigationTracker.getViewName(true));
+  }
+
+  public void sendChipTapOnApp(String bundleTag, String packageName, ChipManager.Chip chip) {
```

Comment:
Similarly to other analytics event methods, there could be a method to create the "base map" with the information that is common with all events and this would avoid repeating code.